pyvista/core/pointset.py

```diff
@@ -219,14 +218,14 @@ def scale(
 
         Parameters
         ----------
-        xyz : scale factor list or tuple or np.ndarray
-            Length 3 list, tuple or array.
+        xyz : list or tuple or numpy.ndarray
+            Scale factor in x, y, and z directions. Length 3 list, tuple or
+            array.
 
         transform_all_input_vectors : bool, optional
-            When ``True``, all input vectors are
-            transformed. Otherwise, only the points, normals and
-            active vectors are transformed. This is only valid when not
-            updating in place.
+            When ``True``, all input vectors are transformed. Otherwise, only
+            the points, normals and active vectors are transformed. This is
+            only valid when not updating in place.
```

Comment:
I know this is only technically related to the PR, but I don't like the look of these methods.

`_PointSet.scale()` here says
1. by default only points, normals and active vectors and transformed
2. otherwise everything is, but only when not `inplace`.

But in https://github.com/pyvista/pyvista/pull/2740/files#diff-74eba4a52c2594069f09d6ca675e50579aa1cdf7bb4d2955f44a46482ebb395aL264-L266 we are manually only transforming points (no data!) in case of `inplace`. And I don't see a reason why `inplace` would have to be treated differently, other than this bug of only transforming points manually in case of `inplace`.

So, shouldn't we instead always call `super().scale()` (`DataSet.scale()` -> `DataSetFilters.transform()`) and let it handle inplaceness? Even if not, we still have to transform the active arrays!

Finally, I don't think we need `np.asarray` in `self.points *= np.asarray(xyz)` but that's just a small blemish even if I'm right.

Reply:
Will follow up in another PR. This is outside of this PR's scope.